src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/states/State.kt

```diff
@@ -0,0 +1,4 @@
+package io.github.fourlastor.wilds_launcher.states
+
+interface State {
+}
```

Comment:
Brackets are optional in kotlin
```suggestion
interface State
```